Update readme.md (#8)
lucasferraco/github-notification-client: github_notification_client/README.md

# Octopus

Keep updated about the last changes in your GitHub.
Octopus is an app that notifies you about the last events in your GitHub

## Getting Started

> clone the repository and run:

Reviewer: ```suggestion
Clone the repository and run:
```

```
flutter packages get
``` 

### Prerequisites

> You need have be flutter installed in your machine.

Reviewer: ```suggestion
You need to have Flutter installed in your machine.
```

> to more information about the installation: https://flutter.dev/docs/get-started/install

Reviewer: ```suggestion
Here's a guide on how to install it: https://flutter.dev/docs/get-started/install
```

### Building and Installing

> Open a emulator or Simulator and run:

Reviewer: ```suggestion
Open an Android emulator or an iOS simulator and run:
```

```
flutter run
```

## Running the tests

To run all tests execute:

```
flutter test
```

## Built With

* [Flutter](https://github.com/flutter/flutter)

## Contributing

Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us. 

## Authors

> * **Lucas Ferraço** - *Initial work* - [lucasferraco](https://github.com/lucasferraco)

Reviewer: I didn't really understand why to say "Initial work". What do you think about the model below?
```suggestion
- [**Lucas Ferraço**](https://github.com/lucasferraco)
```
